generators/server/templates/src/main/java/package/config/_MicroserviceSecurityConfiguration.java

```diff
@@ -121,9 +129,27 @@ public TokenStore tokenStore() {
       @Bean
       public JwtAccessTokenConverter jwtAccessTokenConverter() {
           JwtAccessTokenConverter converter = new JwtAccessTokenConverter();
-          converter.setSigningKey(jHipsterProperties.getSecurity().getAuthentication().getJwt().getSecret());
-
+          converter.setVerifierKey(getKeyFromAuthorizationServer());
           return converter;
       }
+
+      @Bean
+      @Qualifier("LoadBalancedRestTemplate")
```

Comment:
I alsso think you can remove the @Qualifier if you call the method "public RestTemplate  loadBalancedRestTemplate(RestTemplateCustomizer customizer)"